flavors/swagger-ui-react/index.jsx

```diff
@@ -102,7 +103,8 @@ SwaggerUI.propTypes = {
   defaultModelsExpandDepth: PropTypes.number,
   presets: PropTypes.arrayOf(PropTypes.func),
   deepLinking: PropTypes.bool,
-  tryItOutEnabled: PropTypes.bool
+  tryItOutEnabled: PropTypes.bool,
+  displayRequestDuration: PropTypes.bool,
 }
 
 SwaggerUI.defaultProps = {
```

Comment:
Given `displayRequestDuration` is not required prop, what about giving it a default value of `false` in defaultProps?

Reply:
@char0n done